sympy/physics/continuum_mechanics/beam.py

```diff
@@ -13,6 +13,16 @@
 from sympy.integrals import integrate
 from sympy.series import limit
 from sympy.plotting import plot, PlotGrid
+from sympy.external import import_module
+from sympy.utilities.decorator import doctest_depends_on
+from sympy import lambdify
+from sympy.core.compatibility import iterable
+
+matplotlib = import_module('matplotlib', __import__kwargs={'fromlist':['pyplot']})
+numpy = import_module('numpy', __import__kwargs={'fromlist':['linspace']})
+
+__doctest_requires__ = {('Beam.plot_loading_results',): ['matplotlib']}
```

Comment:
@oscargus Since `plot_loading_results()` now uses `PlotGrid`, it no longer requires matplotlib and numpy to be imported.
So do we really need these lines here?

Reply:
In any case `__doctest_requires__` is used by doctest-plus but I don't think that SymPy's homegrown doctest runner uses it.

Reply:
Maybe not. I was primarily focusing on resolving the conflicts, so must admit that I didn't really check what was needed that carefully. Partly as I did it directly in the browser on Github.